base/applications/osk/main.c

```diff
@@ -57,6 +58,60 @@ int OSK_SetImage(int IdDlgItem, int IdResource)
     return TRUE;
 }
 
+/***********************************************************************
+ *
+ *          OSK_WarningProc
+ *
+ *  Function handler for the warning dialog box on startup
+ */
+INT_PTR CALLBACK OSK_WarningProc(HWND hDlg, UINT Msg, WPARAM wParam, LPARAM lParam)
+{
+    /* Declare the icon variable (and initialize it on WM_INITDIALOG) */
+    HICON hIcon;
```

Comment:
This hIcon's value will be lost when the control got out of the function. Use a static variable or a global variable to keep the value.